src/components/home/elements/WithdrawToken.vue

```diff
@@ -0,0 +1,763 @@
+            type: BigNumber,
+            default: function() {
+                return BigNumber(0)
+            },
+            require: true
+        },
+        balance: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        tokenUnity: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        coldAddresses: {
+            type: Object,
+            default: function() {},
+            require: true// Can not be removed ,cause this is an inherited property may be used somewhere
+        },
+        addresses: {
+            type: Object,
+            default: function() {},
+            require: true
+        },
+        maxSupply: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        currentSupply: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        walletType: {
+            type: String,
+            default: 'hotWallet',
+            require: true
+        },
+        tokenId: {
+            type: String,
+            default: '',
+            require: true
+        },
+        issuer: {
+            type: String,
+            default: '',
+            require: true
+        },
+        address: {
+            type: String,
+            default: ''
+        }
+    },
+    computed: {
+        contractId() {
+            return transaction.tokenIDToContractID(this.tokenId)
+        },
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        defaultColdAddress() {
+            if (this.noColdAddress) return ''
+            return Object.keys(this.coldAddresses)[0]
+        },
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.defaultAddress))
+        },
+        secretInfo() {
+            return JSON.parse(
+                seedLib.decryptSeedPhrase(this.userInfo.info, Vue.ls.get('pwd')))
+        },
+        seedPhrase() {
+            return seedLib.decryptSeedPhrase(this.secretInfo.encrSeed, Vue.ls.get('pwd'))
+        },
+        wordList() {
+            return this.seedPhrase.split(' ')
+        },
+        noColdAddress() {
+            return Object.keys(this.coldAddresses).length === 0 && this.coldAddresses.constructor === Object
+        },
+        dataObject() {
+            return {
+                protocol: PROTOCOL,
+                api: this.coldApi(),
+                opc: OPC_FUNCTION,
+                address: this.address,
+                // senderPublicKey: this.coldAddresses[this.address].publicKey,
+                fee: this.fee * VSYS_PRECISION,
+                feeScale: FEE_SCALE,
+                timestamp: Date.now(),
+                attachment: '',
+                contractId: this.contractId,
+                functionId: WITHDRAW_FUNCIDX,
```

Comment:
Only set `functionId` to constant is wrong. `functionId` will be changed for different type of token. (For split supported token, functionId is 7. But split unsupported token, it is 6)